docs/source/literal_types.rst

```diff
@@ -138,8 +138,27 @@ the above program almost as if it were written like so:
     reveal_type(19)
     expects_literal(19)
 
-This is why ``expects_literal(19)`` type-checks despite the fact that ``reveal_type(c)``
-reports ``int``.
+In other words, the type of ``c`` is *context-dependent*: It could be either ``int``
```

Comment:
This sentence is a bit repetitive, there is almost the same one paragraph above, but it uses term "context-sensitive" instead of "context-dependent". It is better to be more consistent with terms here.

Reply:
I tried editing this section of the docs more heavily. If you have time, could you give the new version a second sanity check?